dvc/commands/repro.py

```diff
@@ -135,6 +136,15 @@ def add_arguments(repro_parser):
             "from the run-cache."
         ),
     )
+    repro_parser.add_argument(
+        "--allow-missing",
+        action="store_true",
+        default=False,
+        help=(
+            "Allows to skip stages with no other changes than missing data. "
+            "Useful to be combined with `--pull`."
```

Comment:
```suggestion
            "Skip stages with missing data but no other changes."
```

Minor suggestions. Took out "Useful to be combined with `--pull`" because I don't think it's obvious on its own and explaining is probably too much text for the CLI help. I'd suggest leaving it for the docs.